test/lib/not-found.js

```diff
@@ -28,7 +28,8 @@ describe('not-found', () => {
 
       res.render.should.have.been.calledWith('404', {
         title: 'Not found',
-        description: 'There is nothing here'
+        description: 'There is nothing here',
+        startLink: 'not-here'
```

Comment:
For tests like this where we are testing particular properties being passed to the template renderer it's generally better to match against `sinon.match({ title: '...', description: '...' })` than add extra out-of-scope properties.

That way if the test `renders 404 with a default title and description` fails then it's actually because it's not doing the thing that's described, and not because it's received an extra unrelated property, and tests only fail when things are actually wrong.